assets/src/js/page_header_parallax.js

```diff
@@ -0,0 +1,38 @@
+const isSmallWindow = () => window.innerWidth < 992;
```

Comment:
What do you think to nest these local functions into the `setupPageHeaderParallax` scope?

Reply:
Then they would get defined every single time we call the function, whereas here we only define them once 🤔

Reply:
Just in case, I meant this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions#nested_functions_and_closures. We could keep it for now!